Story/Fireisland/CoreFireisland.cs

```diff
@@ -0,0 +1,240 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Story/Phoenixrise.cs
+
+using Skua.Core.Interfaces;
+
+public class Fireisland
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+    public PhoenixriseStory PhoenixriseStory = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+        Core.RunCore();
+        Core.SetOptions(false);
+    }
+
+        public void CompleteFireIsland()
+        {
+            if (Core.isCompletedBefore(4235))
+            {Core.Logger("You have already completed FireIsland storyline");
+                return;}
+        Embersea();
+        Pyrewatch();
+        PhoenixriseStory.Feverfew();
+        PhoenixriseStory.Phoenixrise();
+        Fireforge();
+        Lavarun();
+        Brimstone();
+        Nightmare();   
+        }
+    public void Embersea()
+    {
+        if (Core.isCompletedBefore(4055))
+        {Core.Logger("You have already completed Embersea storyline");
+            return;}
+
+        Story.PreLoad();
+
+        //Heat of Battle 4054
+        Story.KillQuest(4054, "Embersea", new[] { "Flame Soldier", "Storm Scout" });
+        //Light the Flame 4055
+        Story.MapItemQuest(4055, "Embersea", 3153, 22);
+        Story.KillQuest(4055, "Embersea", "Living Lava");
+        //Kill It With Fire [Member] 4056
+        if (!Core.IsMember)
```

Comment:
just wrap the quest in a members (true) check
if (Core.IsMember)
	Story.KillQuest(4056, "Embersea", new[] { "Coal Creeper", "Pyradon", "Fyresyn" });